content/_secrets.md

```diff
@@ -233,6 +239,7 @@ Property | Type | Description
 `value` | string | The secret's value.
 `update_if_exists` | boolean | Update value if secret already exists (optional, defaults to false).
 `visibility` | string | Optional parameter, defines who can see the secret (default: tenant). **Supported for Cloudify Manager 4.3 and above.**
+`is_hidden_value` | boolean| Optional parameter, determines who can see the secret's value, if True the secret's value is only shown to the user that created the secret and to admins. (default: false). **Supported for Cloudify Manager 4.4 and above.**
```

Comment:
`is_hidden_value` | boolean| Optional parameter, determines who can see the value of the secret. If True, the value of the secret is only shown to the user that created the secret and to admins. (default: false). **Supported for Cloudify Manager 4.4 and above.**